client-java/controller/src/main/java/org/evomaster/client/java/controller/db/SqlScriptRunner.java

```diff
@@ -357,6 +360,29 @@ public static void execScript(Connection conn, String script) throws SQLExceptio
         }
     }
 
+    public static List<String> extractSql(String script){
+        String[] commands = script.split(";");
+        return Arrays.stream(commands).filter(s-> !s.replaceAll("\n","").isEmpty()).map(s-> s+";").collect(Collectors.toList());
+    }
+
+    /**
+     *
+     * @param commands
+     * @return
+     */
+    public static  Map<String, Set<String>> extractSqlTableMap(List<String> commands){
```

Comment:
add more JavaDoc. eg, what it needs in input, and what does it return (what is the key in the map? what are the values in the Set? etc)